azure-pipelines.yml

```diff
@@ -103,6 +103,15 @@ jobs:
     configuration: Release
     testArguments: -testCoreClr
 
+- template: eng/pipelines/test-windows-job.yml
+  parameters:
+    testRunName: 'Test Windows CoreCLR IOperation Debug'
+    jobName: Test_Windows_CoreClr_IOperation_Debug
+    buildJobName: Build_Windows_Debug
+    testArtifactName: Transport_Artifacts_Windows_Debug
+    configuration: Debug
+    testArguments: -testCoreClr -testIOperation -testCompilerOnly
```

Comment:
did you observe a difference in the number of tests run between `-testCoreClr` and `-testCoreClr -testCompilerOnly`?

Reply:
Yes, looking at the pipeline it runs about 4k less tests.